src/components/reusable/datePicker/datepicker.scss

```diff
@@ -1,28 +1,135 @@
 @use '../../../common/scss/global.scss';
 @use '../../../common/scss/form-input.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/visibility.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
 
 :host {
-  display: inline-block;
+    display: inline-block;
+    width: 100%;
```

Comment:
We don't set a default width on any of the other form inputs, usually just let the application control it.

Reply:
✅